frame/glutton/src/tests.rs

```diff
@@ -86,52 +89,81 @@ fn expand_and_shrink_trash_data_works() {
 #[test]
 fn setting_compute_works() {
 	new_test_ext().execute_with(|| {
-		assert_eq!(Compute::<Test>::get(), Perbill::from_percent(0));
+		assert_eq!(Compute::<Test>::get(), Zero::zero());
 
-		assert_ok!(Glutton::set_compute(RuntimeOrigin::root(), Perbill::from_percent(70)));
-		assert_eq!(Compute::<Test>::get(), Perbill::from_percent(70));
+		assert_ok!(Glutton::set_compute(RuntimeOrigin::root(), FixedU64::from_float(0.3)));
+		assert_eq!(Compute::<Test>::get(), FixedU64::from_float(0.3));
 		System::assert_last_event(
-			Event::ComputationLimitSet { compute: Perbill::from_percent(70) }.into(),
+			Event::ComputationLimitSet { compute: FixedU64::from_float(0.3) }.into(),
 		);
 
 		assert_noop!(
-			Glutton::set_compute(RuntimeOrigin::signed(1), Perbill::from_percent(30)),
+			Glutton::set_compute(RuntimeOrigin::signed(1), FixedU64::from_float(0.3)),
```

Comment:
Maybe better to set a different value than current one, otherwise it would be no-op even if it is set, no ?
```suggestion
			Glutton::set_compute(RuntimeOrigin::signed(1), FixedU64::from_float(0.5)),
```